styleguide.setup.js

```diff
@@ -45,6 +66,11 @@ global.iconOptions = {
   'Resilience-AD': resilienceadSvg,
 }
 
+// Inject the ToastContainer so we can push toast notifications
+let toastContainerDiv = document.createElement('div')
+document.getElementsByTagName('body')[0].append(toastContainerDiv)
+ReactDOM.render(React.createElement(SynapseToastContainer), toastContainerDiv)
```

Comment:
Injecting a single SynapseToastContainer into the Styleguidist page so we can push toasts from demos